leftwm/src/bin/leftwm.rs

```diff
@@ -138,9 +146,10 @@ fn start_leftwm() {
 
     let flag = get_sigchld_flag();
 
-    loop {
+    let mut no_error_occured = true;
+    while no_error_occured {
```

Comment:
Ofc at some point we need to have some negative logic:
```suggestion
    let mut error_occured = false;
    while !error_occured {
```